app/js/actions/encounterType.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+import * as types from './actionTypes';
```

Comment:
For the above line, did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports